content/es/function-as-a-service.md

```diff
@@ -0,0 +1,35 @@
+---
+title: Función como Servicio (FaaS)
+status: Completed
+category: Tecnología
+tags: ["infrastructure", "", ""]
+---
+
+## ¿Qué es?
+
+La Función como Servicio (FaaS según sus siglas en Inglés) es un tipo de [servicio](/service/) [sin servidor](/es/serverless/) en [computación en la nube](/es/cloud-computing/)
+que permite ejecutar código como respuesta a ciertos eventos
+sin necesidad de mantener infraestructura compleja
+típicamente asociada con la creación y lanzamiento de aplicaciones en [arquitectura de microservicios](/es/microservices-architecture/).
+Con FaaS, los usuarios manejan solo las funciones y la información mientras que el proveedor de nube maneja la aplicación.
+Esto permite a los desarrolladores ejecutar las funciones que necesitan sin tener que pagar por los servicios cuando el código no está ejecutándose.
+
+## Problema que aborda
+
+En un escenario tradicional de servidores locales, una organización maneja y mantiene su propio centro de datos.
+Esta organización debe invertir en servidores, almacenamiento, software y otras tecnologías
+y potencialmente contratar un equipo de TI o contratistas para comprar, manejar y actualizar todo el equipamiento y sus licencias.
+El centro de datos tiene que ser construido para soportar grandes demandas, incluso cuando la carga de trabajo se vea disminuida y los equipos se utilizan muy por debajo de su capacidad.
+En caso contrario si la organización crece, el equipo contratado podría no cubrir la demanda.
+Bajo un modelo estándar de computación en la nube basado en [Infraestructura-como-Servicio (IaaS)](/es/infrastructure-as-a-service/),
+los usuarios pueden contratar de antemano unidades de capacidad, es decir, pagando a un proveedor de nube pública por componentes de un servidor que siempre está encendido para ejecutar una serie de aplicaciones.
```

Comment:
```suggestion
los usuarios pueden contratar de antemano unidades de capacidad, es decir, pagando a un proveedor de nube pública por componentes de un servidor que siempre está encendido para ejecutar tus aplicaciones.
```

Reply:
@electrocucaracha la palabra `tus` no califica como lenguaje coloquial? ([a evitar](https://github.com/cncf/glossary/blob/main/content/es/style-guide/_index.md)).

Reply:
Sería bueno buscar un punto intermedio, ya que la [versión en inglés](https://github.com/cncf/glossary/blob/main/content/en/function-as-a-service.md?plain=1#L25) utiliza `your`.

Reply:
Igual opino que manejar en primera persona `tú/tus` es lo que podríamos manejar siguiendo la misma convención de la documentación original en inglés `you/your`